x-pack/plugins/observability_onboarding/e2e/cypress/e2e/home.cy.ts

```diff
@@ -13,6 +13,7 @@ describe('[Observability onboarding] Landing page', () => {
   describe('Entry point', () => {
     it('when clicking on the logs card the user is navigated to the observability onboarding page', () => {
       cy.getByTestSubj('guideButtonRedirect').click();
+      cy.getByTestSubj('guide-filter-observability').click();
```

Comment:
I have absolutely no idea why this was broken on my PR and not on main. It seems like a regression introduced by https://github.com/elastic/kibana/pull/171586, but no clue why we're not seeing it on main... Anyway...

Reply:
As we discussed on Slack, unfortunately, I don't think this test suite was run on https://github.com/elastic/kibana/pull/171586 since changes weren't made to this plugin. 

I took a closer look and I think this test change is OK. The landing page no longer displays all cards by default, so the test setup will have to first click the filter before the logs card renders.